<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listOrder">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>661</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listV">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>661</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ol">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>93</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>93</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editTodo">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editV">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>380</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
